cmd/buildd/main_containerd.go

```diff
@@ -1,25 +1,74 @@
-// +build containerd,!standalone
+// +build containerd
 
 package main
 
 import (
-	"github.com/moby/buildkit/control"
+	"os"
+	"strings"
+
+	"github.com/moby/buildkit/worker"
+	"github.com/moby/buildkit/worker/containerd"
+	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
 )
 
-func appendFlags(f []cli.Flag) []cli.Flag {
-	return append(f, []cli.Flag{
+func init() {
+	appFlags = append(appFlags,
+		cli.StringFlag{
+			Name:  "containerd-worker",
+			Usage: "enable containerd workers (true/false/auto)",
+			Value: "auto",
+		},
+		cli.BoolFlag{
+			Name:  "containerd-worker-multiple-snapshotters",
```

Comment:
`containerd-worker-snapshotter` ? I think we could either show multiple automatically (if only one works) or let user specify them all as different workers. This boolen flag doesn't seem very useful. Could just allow one atm and leave this to follow-up.